tests/js/contentAware/includes.js

```diff
@@ -1,11 +1,12 @@
 "use strict";
 
 // The base classes we are testing
-require("../../../index.js");
+var express = require("../../../index.js");
```

Comment:
This variable is confusingly named with respect to others - for example in static.js you have a top-level express which represents the real express rather than gpii-express. I think this would be cleaner as

require("../../../index.js") // or, current infusion allows you to write fluid.require("%gpii-express") using the new "pre-inspection" system
var gpii = fluid.registerNamespace("gpii");

then - 

gpii.express.loadTestingSupport();